tests/lib/gladecheck.py

```
@@ -78,107 +56,16 @@ def checkGlade(self, glade_tree):
         """
         pass
 
-    def test_glade_file(self):
+    def check_glade_files(self):
         """Run checkGlade for each glade file."""
-        for tree in self.glade_trees:
+        for tree in glade_trees:
```

Could you raise an error if there are no trees to check, please?